hi.py

```diff
@@ -0,0 +1,8 @@
+def hello():
+  return "Hello User" 
+
+    
+    
+    
+if name == "__name___"
+    hello()
```

Comment:
<a href="https://www.pantomax.co" target="_blank"><img src="https://raw.githubusercontent.com/pantomaxdotco/assets/refs/heads/main/pr-comment-branding.png" width="50"></a>
Indent the line properly to align with the `if` statement.

```py
    hello()
```